process_pr.py

```diff
@@ -609,10 +610,10 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
     WATCHERS = read_repo_file(repo_config, "watchers.yaml", {})
     # Given the packages check if there are additional developers watching one or more.
```

Comment:
Thanks @smuzaffar , maybe this comment can be updated, e.g.
```suggestion
    # Given the files modified by the PR, check if there are additional developers watching one or more.
```